.github/workflows/ci.yml

```
@@ -131,7 +131,7 @@ jobs:
           {version: '11', experimental: false},
           {version: '17', experimental: false},
           {version: '24', experimental: false},
-          {version: '25.0.0-ea.25.0.ea', experimental: true}]
+          {version: '25-ea', experimental: true}]
```

```suggestion
          {version: '25.0.0-ea.27.0.ea', experimental: true}]
```

Let's keep it pinned to a specific build. We don't know when the next breaking change comes to another build. Based on your comment I'm trying 27, let's see whether that build already exists.